apps/script/HoC2023AiGenerateWeights.py

```diff
@@ -107,41 +111,91 @@
 background_dict = {}
 foreground_dict = {}
 
-# Calculate and print similarity scores
-for emoji_name in emoji_names_list:
-    palette_scores = []
-    for palette_word in palette_model_descriptive_names:
-        id_token = nlp(emoji_name)
-        palette_token = nlp(palette_word)
-        similarity_score = id_token.similarity(palette_token)
-        palette_scores.append(round(similarity_score, 2))
+# Define caches to store ada's raw embedding outputs to reduce query costs
+# These caches are stored as pickle files; python's native way to serialize data
+embedding_paths = ['apps/static/dance/ai/model/input_embeddings.pkl',
+'apps/static/dance/ai/model/palette_embeddings.pkl',
+'apps/static/dance/ai/model/background_embeddings.pkl',
+'apps/static/dance/ai/model/foreground_embeddings.pkl']
+input_path = embedding_paths[0]
```

Comment:
+1 to using a map/dictionary, also are these individual variables used anywhere?